paris/src/main/java/com/airbnb/paris/proxies/ViewProxy.java

```diff
@@ -214,6 +223,33 @@ public void setBackground(@Nullable Drawable drawable) {
         getView().setBackground(drawable);
     }
 
+    @Attr(R2.styleable.Paris_View_android_backgroundTint)
+    public void setBackgroundTint(ColorStateList colorStateList) {
```

Comment:
Is `colorStateList` nullable? Assuming so you can annotate this with `@Nullable` and add a test for it.

Reply:
yes it can be.